src/set.js

```diff
@@ -10,11 +11,13 @@ function set(basePath, input, output, terminal) {
     const folder = path.basename(_basePath)
     const file = path.join(_basePath, '.this')
 
-    const info = fs.existsSync(file) ? JSON.parse(fs.readFileSync(file, 'utf-8')) : { title: folder }
+    const info = fs.existsSync(file) ? JSON.parse(fs.readFileSync(file, 'utf-8')) : {
+        title: folder
+    }
 
     let step = 1
 
-    output.write(`title [${info.title}]: `)
+    output.write(`\n${color.Color_Off}${color.Blue}# ${color.Color_Off}${color.Red}title${color.Color_Off} [${color.Yellow}${info.title}${color.Color_Off}] ~$ `)
```

Comment:
MAY be more convenient to provide a helper script for colouring, instead of doing it directly here